### PR TITLE
Google sign in issue

### PR DESCRIPTION
**Pull request by DamianFalkowski**

### CODE REVIEW

File: BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 if (data === "ok") {
                     location.href = "/Home/Overview";
                 } else if (data === "bad_domain") {
-                    alert("Your google account's domain is not allowed to login using google. You will be logged out from current account. Please try to log in using good account.");
+                    alert("Your google account's domain is not allowed to login using google. You will be logged out from current account. Please try to log in using account allowed to enter.");
```
Comment (b1ker) [inline BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml]: Please revise the statement - it should state somehting like Your xxx is not allowed to use this service.

File: BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -2,17 +2,57 @@
...
+            url: '@Url.Action("GoogleLogin", "Home")',
+            type: 'POST',
+            dataType: 'json',
+            cache: false,
+            data: { token: googleUser.getAuthResponse().id_token },
+            success: function (data) {
+                if (data === "ok") {
+                    location.href = "/Home/Overview";
+                } else if (data === "bad_domain") {
+                    alert("Your google account's domain is not allowed to login using google sign in. You will be logged out from current account. Please try to log in using account allowed to enter.");
```
Comment (b1ker) [inline BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml]: Please revise the statement - it should state somehting like Your ... is not allowed to use this service.

Comment (DamianFalkowski) [inline BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml]: smth like this?
"Your account is not allowed to use this service. You will be logged out. Please try to log in using account allowed to enter."

Comment (DamianFalkowski) [inline BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml]: or just first sentence?

Comment (b1ker) [inline BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml]: what you proposed is ok.

**Pull request merged**